tests/pyspark-notebook/test_spark.py

```diff
@@ -16,3 +16,21 @@ def test_spark_shell(container: TrackedContainer) -> None:
     )
 
     assert "res0: Int = 2" in logs, "spark-shell does not work"
+
+
+def test_pandas_version(container: TrackedContainer) -> None:
+    """Checking if pandas is installed and its version is 1.5.3
+    This is only needed for Spark before 4.0 see #1924"""
+    command = [
+        "start.sh",
+        "bash",
+        "-c",
+        "python -c 'import pandas; print(pandas.__version__)'",
+    ]
+    logs = container.run_and_wait(
+        timeout=60,
+        tty=True,
+        command=command,
+    )
+
+    assert "1.5.3" in logs, "pandas version is not 1.5.3"
```

Comment:
I actually meant trying to run this inside docker image:

```
pdf = pd.DataFrame([("1", "2"), ("0", "3"), ("2", "0"), ("1", "1")], columns=["a", "b"])
psdf = ps.from_pandas(pdf)
self.assert_eq(pdf.cov(), psdf.cov())
```

I'm not sure it will work though.

Reply:
it is enough to test for the pandas version

Reply:
Ok.